system-parachains/coretime/coretime-kusama/Cargo.toml

```
@@ -0,0 +1,202 @@
+[package]
+name = "coretime-kusama-runtime"
+description = "Kusama's Coretime parachain runtime"
+repository.workspace = true
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+build = "build.rs"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
+hex-literal = "0.4.1"
+log = { version = "0.4.20", default-features = false }
+scale-info = { version = "2.9.0", default-features = false, features = ["derive"] }
+serde = { version = "1.0.195", optional = true, features = ["derive"] }
+smallvec = "1.11.1"
+
+# Local
+kusama-runtime-constants = { path = "../../../relay/kusama/constants", default-features = false}
+system-parachains-constants = { path = "../../constants", default-features = false }
+
+# Substrate
+frame-benchmarking = { default-features = false, optional = true, version = "29.0.0" }
+frame-executive = { default-features = false , version = "29.0.0" }
+frame-support = { default-features = false , version = "29.0.0" }
+frame-system = { default-features = false , version = "29.0.0" }
+frame-system-benchmarking = { default-features = false, optional = true , version = "29.0.0" }
+frame-system-rpc-runtime-api = { default-features = false , version = "27.0.0" }
+frame-try-runtime = { default-features = false, optional = true , version = "0.35.0" }
+pallet-aura = { default-features = false, version = "28.0.0" }
```

```suggestion
pallet-aura = { default-features = false, version = "28.0.0", features = ["experimental"] }
```
this should make it compilable